lib/src/ast/event.dart

```diff
@@ -85,14 +85,20 @@ abstract class EventAst implements TemplateAst {
 ///
 /// Clients should not extend, implement, or mix-in this class.
 class ParsedEventAst extends TemplateAst
-    with EventAst, TagOffsetInfo, SpecialOffsetInfo {
-  /// Token representing the `(property)` element decorator.
+    with EventAst, ParsedDecoratorAst, TagOffsetInfo {
```

Comment:
I think this would be cleaner with `implements` than using a mix in, since the mixed in classes don't define any behavior